Fixed compatibility issue with Cython 3.1.1 -- by :user:`bdraco`
